File: Year02/2nd semester/MPP/Lab/mpp-proiect-csharp-AlexandruPaul21/TestApi/TestApi/Flight.cs
namespace model;

[Serializable]
public class Flight : Entity<long>
{
    public string Destination { get; set; }
    public DateTime DepartureDateTime { get; set; }
    public string Airport { get; set; }
    public int AvailableSeats { get; set; }

    public Flight(string destination, DateTime departureDateTime, string airport, int availableSeats)
    {
        Destination = destination;
        DepartureDateTime = departureDateTime;
        Airport = airport;
        AvailableSeats = availableSeats;
    }

    public string ToString()
    {
        return "Flight{" +
               "id=" + Id + 
                "destination='" + Destination + '\'' +
                ", departureDateTime=" + DepartureDateTime +
                ", airport='" + Airport + '\'' +
                ", availableSeats=" + AvailableSeats +
                '}';
    }
}

File: Year02/2nd semester/MPP/Lab/mpp-proiect-java-AlexandruPaul21/JavaProject/src/main/java/org/example/model/Flight.java
package org.example.model;

import java.time.LocalDateTime;

public class Flight extends Entity<Long> {

    private String destination;
    private LocalDateTime departureDateTime;
    private String airport;
    private int availableSeats;

    public Flight(String destination, LocalDateTime departureDateTime, String airport, int availableSeats) {
        this.destination = destination;
        this.departureDateTime = departureDateTime;
        this.airport = airport;
        this.availableSeats = availableSeats;
    }

    public String getDestination() {
        return destination;
    }

    public void setDestination(String destination) {
        this.destination = destination;
    }

    public LocalDateTime getDepartureDateTime() {
        return departureDateTime;
    }

    public void setDepartureDateTime(LocalDateTime departureDateTime) {
        this.departureDateTime = departureDateTime;
    }

    public String getAirport() {
        return airport;
    }

    public void setAirport(String airport) {
        this.airport = airport;
    }

    public int getAvailableSeats() {
        return availableSeats;
    }

    public void setAvailableSeats(int availableSeats) {
        this.availableSeats = availableSeats;
    }

    @Override
    public String toString() {
        return "Flight{" +
                "destination='" + destination + '\'' +
                ", departureDateTime=" + departureDateTime +
                ", airport='" + airport + '\'' +
                ", availableSeats=" + availableSeats +
                '}';
    }
}
